app/views/middleware_server/show.html.haml

```diff
@@ -4,6 +4,7 @@
   = render(:partial => "layouts/performance_async")
 - elsif @showtype == 'main'
   = render :partial => "layouts/textual_groups_generic"
+  = render :partial => 'jdr_reports_list' if @record.middleware_jdr_reports.any?
```

Comment:
I checked the "textual" partials mentioned by khala. Conditionals are in the views. I'll move the `if` there.